client/components/data/query-reader-follows/index.jsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { requestFollows } from 'state/reader/follows/actions';
+
+class QueryReaderFollows extends Component {
```

Comment:
the trick is that some components need to dispatch again on `componentWillReceiveProps` and some don't.